core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -1695,14 +1367,12 @@ object ReassignPartitionsCommand extends Logging {
         opts.bootstrapServerOpt,
```

Comment:
Yes, if `bootstrapServerOpt` is not in the permitted arg list, the check will fail since `bootstrapServerOpt` is not permitted option for this action.